tests/utils_tests/test_feedgenerator.py

```diff
@@ -120,6 +120,15 @@ def test_feed_with_feed_url_gets_rendered_with_atom_link(self):
         self.assertIn('href="/feed/"', feed_content)
         self.assertIn('rel="self"', feed_content)
 
+    def test_enclosures_are_not_required_for_add_item_of_atom(self):
+        # Regression test for #27879
+        feed = feedgenerator.Atom1Feed('title', '/link/', 'descr')
+        feed.add_item('item_title', 'item_link', 'item_description')
+        try:
```

Comment:
I'll update this... try/except isn't needed -- that just makes debugging in the case of a failure more difficult since it hides the traceback.